src/ansiblelint/utils.py

```diff
@@ -604,6 +620,11 @@ def normalize_task_v2(task: Dict[str, Any]) -> Dict[str, Any]:
         del arguments["argv"]
 
     result["action"].update(arguments)
+
+    # check if module invocated with old style
+    old_style_keys = ("local_action",)
+    result = _extract_old_style_keys_from_task(result, sanitized_task, old_style_keys)
```

Comment:
This is kind of cool.

But, we might not need to change any of the utils because of another newish feature: `needs_raw_task`.

If the rule class has `needs_raw_task = True` as a class level variable, then `matchtask` gets passed the raw/unparsed task in `task["__raw_task__"]`.

Would that work here?

Reply:
Thank you! Using `task["__raw_task__"]`  looks smart and enough for us. Let me fix that.